src/Command/Dispatcher/DispatchSettingsCommand.php

```diff
@@ -58,7 +60,12 @@
             try {
                 $this->io->title($project->name());
 
+                if ('doctrine-phpcr-admin-bundle' !== $project->name()) {
```

Comment:
Why this check ?

Reply:
For testing 😂

Reply:
Will remove it later when the sorting is fixed. Have a short appointment now